SpatialGDK/Source/SpatialGDKEditorToolbar/Public/SpatialGDKEditorToolbar.h

```diff
@@ -79,10 +79,10 @@ class FSpatialGDKEditorToolbarModule : public IModuleInterface, public FTickable
 	bool OnIsSpatialNetworkingEnabled() const;
 
 	void GDKEditorSettingsClicked() const;
-	bool IsUnrealNativeNetworkingChecked() const;
+	bool IsNoAutomaticConnectionSelected() const;
```

Comment:
nice! can you do the same change for lines 83, 84, 86, 87 and 88?

so have `IsLocalDeploymentSelected`, `IsCloudDeploymentSelected`, `SelectNoAutomaticConnection`, `SelectLocalDeployment` and `SelectCloudDeployment